### ISSUE TITLE
(tlg0533, tlg2045) Wrongly oriented open quotation marks

### ISSUE THREAD

**Issue by whoopsedesy** [PerseusDL/canonical-greekLit#1290]
There is a convention in some texts of using U+02BD / U+02BC as open/close quotation marks, rather than U+2018 / U+2019 ([rationale here](https://github.com/PerseusDL/tei-conversion-tools/wiki/Greek-Betacode-to-Unicode-Transformations#problems)). These texts (among others) sometimes use U+02BC as both an open and close quotation mark; e.g. at [Callimachus *Hymns* 1.29](https://github.com/PerseusDL/canonical-greekLit/blob/fb4b6145bfe01f1e80b0a7a2624a164513ac5abb/data/tlg0533/tlg015/tlg0533.tlg015.perseus-grc2.xml#L151):
```
ʼΓαῖα φίλη τέκε καὶ σύ· τεαὶ δʼ ὠδῖνες ἐλαφραί.ʼ
```
instead of
```
ʽΓαῖα φίλη τέκε καὶ σύ· τεαὶ δʼ ὠδῖνες ἐλαφραί.ʼ
```
(You might have to zoom in to see the difference.) Compare to [a scan](https://archive.org/details/callimachihymni00moegoog/page/n36/mode/1up).

**Comment by whoopsedesy** [comment]
The following command seems to fix most of the problems. It looks for U+02BC preceded by space or a `>` character (which should be unusual at the end of a quotation or before an apostrophe), and changes it to U+02BD.
```
find data/tlg0533 data/tlg2045 -name '*-grc*.xml' -print0 | xargs -0 perl -CSD -i -p -n -e 's/(\s+|>)\x{02bc}/$1\x{02bd}/gu'
```
However, the command appears to make at least one mistake, so rather than make a pull request, I'll just leave the command so you can check its output.

The mistake I found is:
* [tlg0533.tlg019.perseus-grc2.xml line 47](https://github.com/PerseusDL/canonical-greekLit/blob/fb4b6145bfe01f1e80b0a7a2624a164513ac5abb/data/tlg0533/tlg019/tlg0533.tlg019.perseus-grc2.xml#L167):
  ```diff
  -<lb/>σάμερον αἱ δῶλαι τὰς κάλπιδας ἢ ʼς Φυσάδειαν
  +<lb/>σάμερον αἱ δῶλαι τὰς κάλπιδας ἢ ʽς Φυσάδειαν
  ```
  I am not sure, but it looks like the U+02BC was originally correctly marking an apostrophe, and should not be changed. (This is an unusual case, with an apostrophe at the beginning of a word.)

This is the only mistake I detected among all the changes in the *Dionysiaca* (tlg2045.tlg001) and the *Hymns* (tlg0533.tlg0015–0020), comparing to book scans:
* https://archive.org/details/dionysiaca03nonnuoft
* https://archive.org/details/callimachihymni00moegoog

I did not check the changes in the *Epigrams* (tlg0533.tlg0003) nor the *Fragments* (tlg0533.tlg004).

**Comment by lcerrato** [comment]
@whoopsedesy 
Yes. All quotation marks should be replaced with `<q>` eventually. 

It is not uncommon for the quotation marks to be inconsistently rendered — it is a problem with most scanned texts and persists to an even greater degree in the other collections. I do not find find/replace a good option for this work because of the spacing and character inconsistencies. There is a wide variety of open/close quote renderings in the collections such as bad spacing, curly versus straight marks, things like '' (double single mark) instead of ", and instances where the open/close is entirely missing. 

Not all editors made these changes at the time of CTS conversion as the guidelines were not established. It is also a major issue with the cts/xml when quotations break over sections, so it can become a time consuming task and it requires care and attention to detail. Each text requires an editor to review for balanced marks at the time of conversion to CTS.

It would be preferable to open issues for texts that have bad or incomplete conversion work. There are numerous works that were incompletely converted and reviewed early on in this transition. We know they need more attention for things like this, and we know that if something like this was not done, then there is probably much more additional review that these works require.

**Comment by whoopsedesy** [comment]
Thanks. That's good to know about `<q>`.

I'll close this issue, then, in favor of comprehensive review of these texts.
